packages/client/src/components/post-form-attaches.vue

```diff
@@ -179,7 +178,7 @@ export default defineComponent({
 		}
 	}
 
-	> .remain {
+	> .count {
```

Comment:
This shouldn't cause a problem because the style is scoped.
https://github.com/outloudvi/misskey/blob/32ec42a89f0ae6d40e047f84e1bef2c0f6820551/packages/client/src/components/post-form-attaches.vue#L134